### PR TITLE
Fix errors in spotbugs-maven-plugin and maven-checkstyle-plugin

### PR DESCRIPTION
**Pull request by PeterSuna**

### CODE REVIEW

File: src/test/java/io/lighty/yang/validator/check_update_from/RFC6020Test.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
  * terms of the Eclipse Public License v1.0 which accompanies this distribution,
  * and is available at https://www.eclipse.org/legal/epl-v10.html
  */
-package io.lighty.yang.validator.checkUpdateFrom;
+package io.lighty.yang.validator.check_update_from;
```
Comment (vainyksi) [inline src/test/java/io/lighty/yang/validator/check_update_from/RFC6020Test.java:8]: using underscore in java packages is not a good practice (there are just few exceptions when it should be used but those are not applicable here)
please do not use underscores in package names

**Review by vainyksi — CHANGES_REQUESTED** [review]

File: src/test/java/io/lighty/yang/validator/check_update_from/RFC6020Test.java (hunk 1 above)
Comment (PeterSuna) [reply src/test/java/io/lighty/yang/validator/check_update_from/RFC6020Test.java]: Done

File: src/main/java/io/lighty/yang/validator/YangContextFactory.java [hunk 2]
```diff
@@ -150,13 +150,21 @@ private static String readFile(final String path) throws IOException {
         if (recursiveSearch) {
             return iterateYangFilesRecursively(testSourcesDir);
         } else {
-            return Arrays.asList(testSourcesDir.listFiles(YANG_FILE_FILTER));
+            File[] files = testSourcesDir.listFiles(YANG_FILE_FILTER);
+            if (files == null) {
+                return null;
```
[the remark this answers is not in the document]
Comment (PeterSuna) [reply src/main/java/io/lighty/yang/validator/YangContextFactory.java]: Done

File: src/main/java/io/lighty/yang/validator/Main.java [hunk 3]
```diff
@@ -179,7 +181,7 @@ private static String getYangtoolsVersion(Class clazz) {
         if (!classPath.startsWith("jar")) {
             // Class not from JAR
             LOG.error("class not from jar file");
-            System.exit(1);
+            throw new RuntimeException();
```
[the remark this answers is not in the document]
Comment (PeterSuna) [reply src/main/java/io/lighty/yang/validator/Main.java]: Done

**Pull request merged**